src/app/settings/edit-email/page.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+import React, { useState } from 'react';
+import styles from './page.module.css';
+import InputField from '@/components/textbox/textbox';
+import ActiveButton from '@/components/Active-Button/ActiveButton';
+import Image from 'next/image';
+import backIcon from '@/assets/BackArrow.svg';
+import Link from 'next/link';
+
+const Editaccount: React.FC = () => {
+  const userEmail = 'Deerbuttcheeks@gmail.com';
+  const [email, setEmail] = useState(userEmail);
+  const [isEmailMatch, setIsEmailMatch] = useState(false);
+
+  const handleEmailChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value;
+    setEmail(value);
+    setIsEmailMatch(value !== userEmail);
+  };
+  return (
+    <div>
+      <Link href="/settings">
+        <Image
+          priority
+          src={backIcon}
+          alt="Back"
+          className={styles.backButtonImage}
+        />
+      </Link>
+      <div className={styles.mainConatiner}>
+        <h1 className={styles.editAccountHeader}>Edit Account</h1>
+        <div className={styles.formContainer}>
+          <div>
+            <label>Email</label>
+            <InputField
+              type="email"
+              value={email}
+              onChange={handleEmailChange}
+              placeholder="Enter email"
+            />
+
+            <ActiveButton text="Change Password" />
+          </div>
+          <ActiveButton
+            isdisabled={!isEmailMatch}
+            text="Save"
+            onClick={() => {}}
+          />
```

Comment:
![image](https://github.com/user-attachments/assets/c3d358ce-d514-4b4e-ae45-62e1fff68c5f)
The save button is too close to the bottom. This will be an issue, especially for mobile, when toast notifications are eventually added. You just have to give the save button a bottom margin.